src/app/chat/signal_handling.nim

```diff
@@ -4,7 +4,7 @@ import
 proc handleSignals(self: ChatController) =
   self.status.events.on(SignalType.Message.event) do(e:Args):
     var data = MessageSignal(e)
-    self.status.chat.update(data.chats, data.messages, data.emojiReactions, data.communities, data.membershipRequests, data.pinnedMessages, data.activityCenterNotification, data.statusUpdates, data.deletedMessages)
+    #self.status.chat.update(data.chats, data.messages, data.emojiReactions, data.communities, data.membershipRequests, data.pinnedMessages, data.activityCenterNotification, data.statusUpdates, data.deletedMessages)
```

Comment:
@saledjenic not your style :) are you sure to save a comment?

Reply:
@borismelnik yes, we need that line commented out. And that's because that part belongs to the old code base, since we are at the moment when we cannot maintain new and old code in the same time, but we still need old code just for the sake of flow and better understanding a feature, I didn't want to remove that line. So till the end of chat/communities refactoring we will have more and more line/snippets of code commented out, but not removed (of course somewhere we will remove it) and once we refactor complete section we will easily remove entire old code containing in `src/app/chat`.